Woof/Woof/Model/PetOwner.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+
+/**
+ A struct representing a pet owner.
+ This struct conforms to the Identifiable and Codable protocols, allowing it
+ to be easily used in SwiftUI and encoded/decoded from JSON.
+ */
+struct PetOwner: Identifiable, Codable {
```

Comment:
nit: maybe `Owner` will be enough as a name for model?

Reply:
Yes, sure. We have only one type of owner and no extra clarification is needed.